test/expected/plan_expand_hypertable-16.out

```diff
@@ -1217,6 +1217,157 @@ transformation would be out of range
          Filter: (time_bucket('@ 1000 days'::interval, "time") < 'Sat Jan 01 00:00:00 294276 PST'::timestamp with time zone)
 (12 rows)
 
+\qecho time_bucket exclusion with run-time constants
+time_bucket exclusion with run-time constants
+PREPARE P1(text)     AS SELECT * FROM metrics_timestamptz WHERE time_bucket($1::interval, time) > '2023-01-01 UTC'::timestamptz ORDER BY time;
+PREPARE P2(text)     AS SELECT * FROM metrics_timestamp   WHERE time_bucket($1::interval, time) > '2023-01-01 UTC'::timestamptz ORDER BY time;
+PREPARE P3(text)     AS SELECT * FROM metrics_timestamptz WHERE time_bucket($1::interval, time) > '2023-01-01 UTC'::timestamp   ORDER BY time;
+PREPARE P4(text)     AS SELECT * FROM metrics_timestamp   WHERE time_bucket($1::interval, time) > '2023-01-01 UTC'::timestamp   ORDER BY time;
+PREPARE P5(interval) AS SELECT * FROM metrics_timestamptz WHERE time_bucket($1::interval, time) > '2023-01-01 UTC'::timestamptz ORDER BY time;
+PREPARE P6(interval) AS SELECT * FROM metrics_timestamp   WHERE time_bucket($1::interval, time) > '2023-01-01 UTC'::timestamptz ORDER BY time;
+PREPARE P7(interval) AS SELECT * FROM metrics_timestamptz WHERE time_bucket($1::interval, time) > '2023-01-01 UTC'::timestamp   ORDER BY time;
+PREPARE P8(interval) AS SELECT * FROM metrics_timestamp   WHERE time_bucket($1::interval, time) > '2023-01-01 UTC'::timestamp   ORDER BY time;
+SET plan_cache_mode TO 'force_custom_plan';
+:PREFIX EXECUTE P1('60 mins');
+                    QUERY PLAN                    
+--------------------------------------------------
+ Custom Scan (ChunkAppend) on metrics_timestamptz
+   Order: metrics_timestamptz."time"
+   Chunks excluded during startup: 5
+(3 rows)
+
+:PREFIX EXECUTE P2('60 mins');
+                   QUERY PLAN                   
+------------------------------------------------
+ Custom Scan (ChunkAppend) on metrics_timestamp
+   Order: metrics_timestamp."time"
+   Chunks excluded during startup: 5
+(3 rows)
+
+:PREFIX EXECUTE P3('60 mins');
+                    QUERY PLAN                    
+--------------------------------------------------
+ Custom Scan (ChunkAppend) on metrics_timestamptz
+   Order: metrics_timestamptz."time"
+   Chunks excluded during startup: 5
+(3 rows)
+
+:PREFIX EXECUTE P4('60 mins');
+                   QUERY PLAN                   
+------------------------------------------------
+ Custom Scan (ChunkAppend) on metrics_timestamp
+   Order: metrics_timestamp."time"
+   Chunks excluded during startup: 5
+(3 rows)
+
+:PREFIX EXECUTE P5('60 mins');
+           QUERY PLAN           
+--------------------------------
+ Sort
+   Sort Key: "time"
+   ->  Result
+         One-Time Filter: false
+(4 rows)
```

Comment:
This query has only immutable operators, so with forced custom plans it uses plan-time exclusion. Same for P8.